pom.xml

```diff
@@ -136,10 +136,15 @@
         <janus.version>0.5.2</janus.version>
         <gremlin.version>3.4.8</gremlin.version>
         <netty.version>4.1.52.Final</netty.version>
+
         <spring-boot.version>2.3.3.RELEASE</spring-boot.version>
         <spring.version>5.2.9.RELEASE</spring.version>
         <spring-security.version>5.4.0</spring-security.version>
         <spring-data.version>2.3.4.RELEASE</spring-data.version>
+
+        <servo-core.version>0.10.1</servo-core.version>
```

Comment:
> this is feb 2016. current is 0.13.0

this is because of some dependency chained issues, this version `0.10.1` solves for some version conflicts in maven. you can test it by removing the `servo-core` depdendency and see what happens on build time